### PR TITLE
Move ProcInfo to facilitate code sharing

### PR DESCRIPTION
**Pull request by rmars**
Now that https://github.com/BuoyantIO/linkerd/pull/189 has merged into master, put #191 on master by running `git rebase --onto master rmars/dashboard-endpoint `

Fixes #191

### CODE REVIEW

**Comment by klingerf** [comment]
:star: :balloon: Looks good to me.

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  function render(data) {
+    var json = $.parseJSON(data);
+    _(json).each(function(obj) {
+      var id = obj.name.replace(/\//g, "-");
+      var value = pretty(obj.name, obj.value);
+      $("#"+id).text(value);
+    });
+  }
```
Comment (esbie) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js]: I don't think this needs to be done now, but it'd be nice if in the future this used process_info.template

**Comment by esbie** [comment]
:star: let the code re-use begin!

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js (hunk 1 above)
Comment (rmars) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js]: Yes!  will do in a follow up branch.  Thanks!

**Pull request merged**